packages/@aws-cdk/aws-quickstarts/README.md

```diff
@@ -1,4 +1,11 @@
 ## AWS Quickstarts for the CDK
+<div class="stability_label"
+     style="background-color: #EC5315; color: white !important; margin: 0 0 1rem 0; padding: 1rem; line-height: 1.5;">
+  Stability: 1 - Experimental. This API is still under active development and subject to non-backward
+  compatible changes or removal in any future version. Use of the API is not recommended in production
+  environments. Experimental APIs are not subject to the Semantic Versioning model.
+</div>
+
 
 This module has been deprecated. Use `@aws-cdk/aws-cloudformation.CfnStack` instead.
```

Comment:
This module has been actually deprecated... 

there are a couple of others. I expected pkglint to enforce that if "deprecated" is set in package.json then stability must be also deprecated (but I guess that's going to be enforce by jsii... still... wouldn't hurt).

Reply:
Oops - an oversight :)